### PR TITLE
Don't use jcenter() as a repository

### LINKED ISSUE #988: Migrate away from JCenter
As per [recent announcement](https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/) JCenter will be shut down at 30 Apr 2021.

We need to refactor plugin build scripts, check documentation and functional tests.

### PR DESCRIPTION
**Pull request by orzeh**
Fixes #988

### CODE REVIEW

**Review by cdancy — APPROVED** [review]
LGTM

File: src/functTest/groovy/com/bmuschko/gradle/docker/tasks/image/DockerListImagesFunctionalTest.groovy [hunk 1]
```diff
@@ -63,6 +64,7 @@ class DockerListImagesFunctionalTest extends AbstractGroovyDslFunctionalTest {
 
                 onNext { images ->
                     if(!images.every { image -> image.repoTags.contains("${IMAGE_ID}:latest") }) {
+                        logger.error("Retuned images: {}", images)
```
Comment (bmuschko) [inline src/functTest/groovy/com/bmuschko/gradle/docker/tasks/image/DockerListImagesFunctionalTest.groovy:67]: Typo in "returned".

**Review by bmuschko — CHANGES_REQUESTED** [review]

**Comment by bmuschko** [comment]
Looks like one of the tests is failing though I don't think it's related to your change. Would you mind having a look either way? https://scans.gradle.com/s/iambqfxgdtsto/tests/:functionalTest/com.bmuschko.gradle.docker.tasks.image.DockerListImagesFunctionalTest/can%20list%20images%20with%20image%20name%20filter#1

**Comment by orzeh** [comment]
@bmuschko yes I've already look into it hence that additional log statement. Unfortunately I have no idea what's wrong - [the output](https://scans.gradle.com/s/lyqzg2oabv6pc/tests/:functionalTest/com.bmuschko.gradle.docker.tasks.image.DockerListImagesFunctionalTest/can%20list%20images%20with%20image%20name%20filter?anchor=e30&focused-exception-line=0-23#1) doesn't make any sense. I think it might be GitHub Actions quirk.

**Comment by bmuschko** [comment]
@orzeh From what I can see in the output of the test and the logic we have in place there, something seems off.

Output is: `repoTags=[busybox:musl]`
The value we compare it with is: `gradle/16b0d6a166bd426b9c41057dfdc80ac9:latest`

**Comment by bmuschko** [comment]
@orzeh Could you try to fix the test? We won't be able to merge otherwise. Or we'll have to ignore the test and fix it later but to be honest, we might as well just delete the test then as nobody will look at it.

**Comment by bmuschko** [comment]
Thanks for the PR. I made the change on `master` as there were additional tests to be fixed.

**Pull request closed without merging**